app/models/UserAnswers.scala

```diff
@@ -73,7 +73,7 @@ object UserAnswers {
         (__ \ "lastUpdated").read[Instant] and
         (__ \ "_id").read[UUID] and
         (__ \ "departureId").readNullable[String] and
-        (__ \ "isTransitional").readWithDefault[Boolean](true)
+        (__ \ "isTransitional").readWithDefault[Boolean](false)
```

Comment:
This should still be true. We know that a document without this field present must have been made during the transition phase.